docs/examples/ambassador-ingress.md

```diff
@@ -1,186 +1,181 @@
-```
-
-Once running, export the default k0s configuration file using
-
-```sh
-docker exec k0s k0s default-config > k0s.yaml
-```
-
-and export the cluster config so you can access it with kubectl:
-
-```sh
-docker exec k0s cat /var/lib/k0s/pki/admin.conf > k0s-cluster.conf
-export KUBECONFIG=$KUBECONFIG:<absolute path to k0s-cluster.conf>
-```
-(somewhat brute-force but gets the job done)
-
-## Configuring k0s.yaml
-
-Open the file in your favorite code editor and add the following extensions at the bottom:
-
-```yaml
-extensions:
-  helm:
-    repositories:
-      - name: datawire
-        url: https://www.getambassador.io
-      - name: bitnami
-        url: https://charts.bitnami.com/bitnami
-    charts:
-      - name: ambassador
-        chartname: datawire/ambassador
-        version: "6.5.13"
-        namespace: ambassador
-        values: |2
-          service:
-            externalIPs:
-            - 172.17.0.2
-      - name: metallb
-        chartname: bitnami/metallb
-        version: "1.0.1"
-        namespace: default
-        values: |2
-          configInline:
-            address-pools:
-            - name: generic-cluster-pool
-              protocol: layer2
-              addresses:
-              - 172.17.0.2
-```
-(you might need to replace the 172.17.0.2 IP with your local IP which you can find higher up in the generated file
-under spec.api.address)
-
-As you can see it adds both Ambassador and Metallb (required for LoadBalancers) with corresponding repositories
-and (minimal) configurations. This example only uses your local network - providing a range of IPs for 
-MetalLB that are addressable on your LAN is suggested if you want to access these services from anywhere on 
-your network.
-
-Now stop/remove your k0s container with `docker stop k0s` and `docker rm k0s`, then start it again with additional ports and the above config file mapped into it:
-
-```sh
-docker run --name k0s --hostname k0s --privileged -v /var/lib/k0s -v <path to k0s.yaml file>:/k0s.yaml -p 6443:6443 -p 80:80 -p 443:443 -p 8080:8080 docker.io/k0sproject/k0s:latest
-```
-
-Let it start, and eventually (this can take some time) you'll be able to list the Ambassador Services:
-
-```shell
-kubectl get services -n ambassador
-NAME                          TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
-ambassador-1611224811         LoadBalancer   10.99.84.151    172.17.0.2    80:30327/TCP,443:30355/TCP   2m11s
-ambassador-1611224811-admin   ClusterIP      10.96.79.130    <none>        8877/TCP                     2m11s
-ambassador-1611224811-redis   ClusterIP      10.110.33.229   <none>        6379/TCP                     2m11s
-```
-
-Install the Ambassador [edgectl tool](https://www.getambassador.io/docs/latest/topics/using/edgectl/edge-control/) 
+# Install Ambassador Gateway on k0s
+
+You can configure k0s with the [Ambassador API
+Gateway](https://www.getambassador.io/products/api-gateway/) and a [MetalLB
```

Comment:
Remove the manual line changes (new line characters) from the middle of the paragraphs (this is general comment for all changes). Otherwise it becomes difficult to modify the text later on and you’d need to change all the line changes to different place. Line changes can be left to the text editor and browser, which will automatically scale the text according to the screen size. The whole docs has been written without manual line changes.